diesel/src/query_builder/mod.rs

```diff
@@ -120,6 +120,9 @@ pub(crate) use self::insert_statement::ColumnList;
 #[cfg(feature = "postgres_backend")]
 pub use crate::pg::query_builder::only::Only;
 
+#[cfg(feature = "postgres_backend")]
+pub use crate::pg::query_builder::tablesample::{Tablesample, TablesampleMethod};
```

Comment:
```suggestion
pub use crate::pg::query_builder::tablesample::{Tablesample};
```

Exposing the `TablesampleMethod` trait shouldn't be required for the user, right?

For the macros we usually expose it via `diesel::internals::macro_name::TablesampleMethod` where macro_name corresponds to the macro requiring that trait.